src/Cli/dotnet/commands/dotnet-sdk/check/RuntimeOutputWriter.cs

```diff
@@ -39,24 +40,31 @@ public void PrintRuntimeInfo()
 
         private string GetRuntimeStatusMessage(NetRuntimeInfo runtime)
         {
-            if (BundleIsEndOfLife(runtime))
+            try
             {
-                return string.Format(LocalizableStrings.OutOfSupportMessage, $"{runtime.Version.Major}.{runtime.Version.Minor}");
+                if (BundleIsEndOfLife(runtime))
+                {
+                    return string.Format(LocalizableStrings.OutOfSupportMessage, $"{runtime.Version.Major}.{runtime.Version.Minor}");
+                }
+                else if (BundleIsMaintenance(runtime))
+                {
+                    return string.Format(LocalizableStrings.MaintenanceMessage, $"{runtime.Version.Major}.{runtime.Version.Minor}");
+                }
+                else if (NewerRuntimePatchExists(runtime))
+                {
+                    return string.Format(LocalizableStrings.NewPatchAvailableMessage, NewestRuntimePatchVersion(runtime));
+                }
+                else
+                {
+                    return LocalizableStrings.BundleUpToDateMessage;
+                }
             }
-            else if (BundleIsMaintenance(runtime))
+            catch (InvalidOperationException)
             {
-                return string.Format(LocalizableStrings.MaintenanceMessage, $"{runtime.Version.Major}.{runtime.Version.Minor}");
-            }
-            else if (NewerRuntimePatchExists(runtime))
-            {
-                return string.Format(LocalizableStrings.NewPatchAvailableMessage, NewestRuntimePatchVersion(runtime));
-            }
-            else
-            {
-                return LocalizableStrings.BundleUpToDateMessage;
+                return "No version found to check if this is up to date. Please check for updates manually.";
```

Comment:
Message probably should be localized

Reply:
Done, I was only able to sadly translate one with poedit, so the others will need to be translated by someone else (after verifying the translated one that I was able to do).

Reply:
Interesting, it thinks to manually updated xlf's are outdated.

Reply:
I decided to instead have OneLocBuild (or whatever it is called) do all of the translations instead for me as it seems that it can automatically run right after the time this would be merged anyway.